addon/components/sl-menu.js

```diff
@@ -338,13 +341,29 @@ export default Ember.Component.extend({
     children: null,
 
     /**
-     * Additional class names to be added to the classNames attribute
+     * Array of classes to be added to the element's class attribute
      *
      * @type {String[]}
      */
     extraClassNames: [],
 
     /**
+     * Additional class string to be added to the element's class attribute
+     *
+     * @type {String}
+     */
+    extraClassNamesString: function() {
```

Comment:
This should be in the "Methods" section labeled as a @function with @returns {String}